microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -313,3 +349,49 @@ def _gpu_provision_check(min_pci_count: int, node: Node, log: Logger) -> None:
     assert_that(len(curr_gpu)).described_as(
         "GPU PCI device count should be same after stop-start"
     ).is_equal_to(len(init_gpu))
+
+
+def __remove_sources_added_by_extension(
+    node: Node, sources_before: List[str], sources_after: List[str]
+) -> None:
+    rm_sources = [source for source in sources_after if source not in sources_before]
+    for source in rm_sources:
+        node.execute(f"rm /etc/apt/sources.list.d/{source}", sudo=True)
+
+
+def __install_driver_using_platform_feature(
```

Comment:
Make this logic in feature level, and change the default value of `install_by_platform` to False.

```python
class GpuSettings(schema.FeatureSettings):
    type: str = FEATURE_NAME_GPU
    install_by_platform: bool = True
```

Reply:
done